### DISCUSSION TITLE
"Arctic Code Vault Contributor" doesn't show my contribution of others OSS, but only my own repo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22087] [New to GitHub]
<p>As Arctic Code Vault Contributor opened, many friends have showed me their contribution to big OSS.<br>
Well, I also did some contribution to others OSS, however, my Arctic Code Vault Contributor badge only shows my own repository contribution.</p>
<p>It’s kinda shame…lol, it’s joke, but why only I can’t get the badge show contribution to others OSS?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi :wave: welcome!</p>
<p>In order for a public repository to be archived in the vault it had to have met the following conditions:</p>
<ul>
<li>Have commits between 11/13/2019 and 02/02/2020</li>
<li>Have commits from the year before the snapshot (02/03/2019 - 02/02/2020) and at least one star</li>
<li>Have 250 or more stars</li>
</ul>
<p>If you’ve contributed to repositories that met that criteria at the time of the snapshot, please open a <a href="https://support.github.com/contact">support ticket</a> and we can look into that for you. Be sure to include the specific commits you believe are missing!</p>

**Comment by ghost** [comment]
<p>ohhh before 02/02/2020 was the cause of my case.<br>
Thank you very much for tellig me!</p>
